docs/appendix/changelog.md

```diff
@@ -21,15 +40,42 @@ date: "2024-09-18"
 
 <!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->
 
-{{% changelog date="2024-11-05" color="added" title="TFLite model moved to module" %}}
-
-The `TFLite CPU` ML model service has moved to a [module](https://app.viam.com/module/viam/tflite_cpu).
-
-{{% /changelog %}}
-
-{{% changelog date="2024-10-28" color="added" title="Deprecate builtin pi model" %}}
-
-The built-in `pi` sensor and board models have moved to the [`raspberry-pi` module](https://github.com/viam-modules/raspberry-pi).
+{{% changelog date="2024-11-12" color="added" title="Builtin models moved to modules" %}}
+
+The following resource models have moved to modules.
+
+| Resource | Model |
+| -------- | ----- |
+| board | [`customlinux`](https://github.com/viam-modules/customlinux/) |
+| board | [`jetson`](https://github.com/viam-modules/nvidia) |
+| board | [`pca9685`](https://github.com/viam-modules/pca/tree/main) |
+| board | [`odroid`](https://github.com/viam-modules/hardkernel) |
+| board | [`ti`](https://github.com/viam-modules/texas-instruments) |
+| board | [`pi`](https://github.com/viam-modules/raspberry-pi) |
+| board | [`orange-pi`](https://github.com/viam-modules/orange-pi) |
+| board | [`upboard`](https://github.com/viam-modules/up) |
+| gripper | [`softrobotics`]() |
+| motor | [`encoded-motor`]() |
+| motor | [`gpiostepper`]() |
+| motor | [`roboclaw`]() |
+| motor | [`tmc5072`](https://github.com/viam-modules/analog-devices) |
+| motor | [`28byj-48`](https://github.com/viam-modules/uln2003) |
+| encoder | [`ams-as5048`](https://github.com/viam-modules/ams/) |
+| movement sensor | [`adxl345`](https://github.com/viam-modules/analog-devices) |
+| movement sensor | [`dual-gps-rtk`](https://github.com/viam-modules/gps/) |
+| movement sensor | [`gps-nmea-rtk-pmtk`](https://github.com/viam-modules/gps/) |
+| movement sensor | [`gps-nmea-rtk-serial`](https://github.com/viam-modules/gps/) |
+| movement sensor | [`gps-nmea`](https://github.com/viam-modules/gps/) |
+| movement sensor | [`imu-wit`](https://github.com/viam-modules/wit-motion) |
+| movement sensor | [`imu-wit-hwt905`](https://github.com/viam-modules/wit-motion) |
+| movement sensor | [`mpu6050`](https://github.com/viam-modules/tdk-invensense) |
+| power sensor | [`ina219`](https://github.com/viam-modules/texas-instruments) |
+| power sensor | [`ina226`](https://github.com/viam-modules/texas-instruments) |
+| sensor | [`bme280`](https://github.com/viam-modules/bosch) |
+| sensor | [`ds18b20`]() |
```

Comment:
And this one?

Reply:
also in my namespace instead of the viam namespace